### PR TITLE
ESLint 9

### LINKED ISSUE #9476: Migration ESLint 9.0.0
https://eslint.org/docs/latest/use/configure/migration-guide

Waiting 
- [x] https://github.com/typescript-eslint/typescript-eslint/issues/8211

Fix https://github.com/jhipster/jhipster-lite/pull/9481

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #9476

### CODE REVIEW

**Comment by qmonmert** [comment]
@pascalgrimaud & @murdos if you have time, maybe be try locally 🙂

**Comment by murdos** [comment]
> @pascalgrimaud & @murdos if you have time, maybe be try locally 🙂

I won't be able to test before one week, but good Idea 🙂

File: .eslintrc.cjs [hunk 1]
```diff
@@ -1,17 +0,0 @@
-module.exports = {
-  env: {
-    node: true,
-  },
-  extends: ['eslint:recommended', 'plugin:vue/vue3-recommended', '@vue/eslint-config-typescript', '@vue/prettier'],
-  parserOptions: {
-    parser: '@typescript-eslint/parser',
-  },
-  plugins: ['pug'],
```
[the remark this answers is not in the document]
Comment (qmonmert) [reply .eslintrc.cjs:9]: no, this command lints pug files `"lint": "eslint . --fix --plugin eslint-plugin-pug src/main/style/**/*.pug && npm run lint:sass && npm run prettier:format",`

**Comment by pascalgrimaud** [comment]
@qmonmert : are you confident enough to merge this?
Is it only for the engine, right?
What about the generated apps?

**Comment by qmonmert** [comment]
@pascalgrimaud yes it's for the engine, I will do generated apps after (there is a ticket for that https://github.com/jhipster/jhipster-lite/issues/9496 ), and I'm confident to merge and if there are some problems I'm ready to fix 🙂

**Comment by qmonmert** [comment]
@pascalgrimaud https://opencollective.com/generator-jhipster/expenses/215615 🙂

**Comment by pascalgrimaud** [comment]
@qmonmert : approved

**Comment by qmonmert** [comment]
@pascalgrimaud thanks

**Pull request merged**